site/content/docs/user/crd.md

```diff
@@ -27,14 +30,16 @@ kamus-cli encrypt
 ```
 Now that you have the data encrypted, create a KamusSecret object, using the following manifest:
 ```
-apiVersion: "soluto.com/v1alpha1"
+apiVersion: "soluto.com/v1alpha2"
 kind: KamusSecret
 metadata:
   name: my-tls-secret     //This will be the name of the secret
   namespace: default      //The secret and KamusSecret live in this namespace
 type: TlsSecret           //The type of the secret that will be created
-data:                     //Put here all the encrypted data, that will be stored (decrypted) on the secret data
+stringData:               //Put here all the encrypted data, that will be stored (decrypted) on the secret data
   key: J9NYLzTC/O44DvlCEZ+LfQ==:Cc9O5zQzFOyxwTD5ZHseqg==
+data:                     //Put here base64 encoded data (usually, binary data like private keys in der format)
```

Comment:
This is encrypted base 64 right?
I think we need to emphasis that.

Reply:
fixed